packages/fx-core/src/plugins/solution/fx-solution/v2/scaffolding.ts

```diff
@@ -96,6 +103,14 @@ export async function scaffoldSourceCode(
         [SolutionTelemetryProperty.ProgrammingLanguage]:
           ctx.projectSetting?.programmingLanguage ?? "",
       });
+    } else {
+      //For SPFx plugin, execute it alone lastly
+      if (SPFxPlugin.scaffoldSourceCode) {
+        const spfxRes = await SPFxPlugin.scaffoldSourceCode(ctx, inputs);
```

Comment:
It seems tricky that you do spfx scaffold after showing scaffold successful notification